src/styles/_functions.scss

```diff
@@ -0,0 +1,4 @@
+@function rem($size) {
+  $remSize: $size / 16px;
```

Comment:
Just curious, why we have to use two units?

Reply:
Since `rem` is more flexible and scalable than `px`, we will use `rem` as our main unit.

Reply:
Normally the design comes with `px` unit instead of `rem`. This converter will make our code more readable. For example, we could write rem(150px) instead of 9.375rem.